apps/client/resources/public/stylesheets/main.css

```diff
@@ -467,6 +469,20 @@ ul#sites-available > li > a {
   line-break: anywhere;
 }
 
+div.missing-keys {
```

Comment:
where possible, I would avoid using px as your unit of measurement. It does not scale well, as compared to a relative unit like `rem` or `em`. 

In this case, I think the pixel is fine for the border radius, but would adjust all the other instances--10px becomes 0.5rem for example.  This makes the site easier to style for phone and desktop.

Reply:
That is a good point!
I've updated it in b701f54